coral-component-button/src/styles/index.styl

```diff
@@ -10,8 +10,27 @@
  * governing permissions and limitations under the License.
  */
 
+@require '../../../coral-theme-spectrum/src/styles/vars.css';
+
 // Avoid FOUC
 button[is="coral-button"] {
   background-color: transparent;
   border-color: transparent;
 }
+
+.coral--light ._coral-ActionButton--quiet:disabled,
+.coral--light ._coral-ActionButton--quiet.is-disabled,
+.coral--light ._coral-Tool:disabled {
```

Comment:
Do we need to update the CSS for `coral-Tool`? Why is it required? Same for all other places.

Reply:
I have revisited the changes.
Yes we do not need to update the css for _coral-Tool.
I have done the changes accordingly.